docs/reference-broker-api.md

```
@@ -0,0 +1,246 @@
+agent-relay up --port 3888                     # local only
+agent-relay up --port 3888 --api-bind 0.0.0.0  # accept remote
+```
+
+All routes below live under `http://<host>:<port>`.
+
+## Authentication
+
+The broker requires an API key on every protected route. Pass it as
+either header:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)